src/test/resources/fixtures/hmrc-frontend/test-fixtures/list-with-actions-empty-list/output.txt

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Just to confirm - when there is an empty list of Actions, should output an empty string?

Reply:
Yes, we have not made any changes to this behavior, so as far as I am aware this is the way it should be working.